.github/workflows/rust.yml

```diff
@@ -89,6 +88,17 @@ jobs:
         run: brew install libtool
         if: runner.os == 'macOS'
 
+      # We cache protoc because it sometimes fails to download, but cache is too slow on Windows.
```

Comment:
Consider creating an upstream issue to add built-in cache to `arduino/setup-protoc` action

Reply:
It has a built-in GitHub and local cache, but we don't use it because it's very slow on Windows:
https://github.com/arduino/setup-protoc/blob/3ea1d70ac22caff0b66ed6cb37d5b7aadebd4623/src/installer.ts#L58